src/test/java/io/reactivex/internal/functions/FunctionsTest.java

```diff
@@ -169,6 +169,54 @@ public Integer apply(Integer t1, Integer t2, Integer t3, Integer t4, Integer t5,
         }).apply(new Object[20]);
     }
 
+    @Test(expected = NullPointerException.class)
+    public void biFunctionFail() throws Exception {
```

Comment:
Please add `@SuppressWarnings({"unchecked", "rawtypes"})` to all these new methods.